using-openspace/ui-panels/geolocation/index.md

```diff
@@ -0,0 +1,81 @@
+---
+authors:
+  - name: Brian Abbott
+    affiliation: American Museum of Natural History
+---
+
+
+# GeoLocation: Pinpoint a Place
+
+:::{figure} toolbar_geolocation.png
+:align: center
+:width: 1000px
+:alt: Toolbar with the Geo Location Panel highlighted
+
+The Geo Location Panel Button in the OpenSpace Toolbar.
+:::
+
+
+Use the GeoLocation Panel to locate places on Earth and fly to them. The panel is straightforward with only few sections: the search area, the Custom Coordinate section, and a list of any added nodes. There is also a section at the top for selecting the anchor (globe) for which to search and/or add nodes for. At the time of writing, only Earth is supported.
+
+:::{figure} geolocation_panel.png
+:align: center
+:width: 400px
+:alt: OpenSpace's GeoLocation Panel
+
+The GeoLocation Panel in OpenSpace.
+:::
+
+## Search
+
+Enter a place or feature on Earth in the search box and hit the {menuselection}`Search` Button. Results will appear below, awaiting your selection.
+
+
+:::{figure} geolocation_panel_search.png
+:align: center
+:width: 1000px
+:alt: OpenSpace's GeoLocation panel searching for Grand Canyon
+
+OpenSpace's GeoLocation Panel with a search for "Grand Canyon."
+:::
+
+For each search result in the list, there are three buttons that result in different actions:
+- ![Fly to button](geolocation_panel_button_flyto.png){w=1em} {menuselection}`Fly To`: Automatically fly to the selected location. A `Cancel` button will appear in the Toolbar to cancel the flight if need be. The duration of the flight will also appear beside the plane icon.
+:::{figure} geolocation_panel_flyto_button.png
+:align: center
+:width: 300px
+:alt: OpenSpace's Stop Recording button
+When OpenSpace is flying to a target automatically, a portion of the Toolbar will change, and the duration of the transition will be shown, 8 seconds in this image, beside a Cancel Button.
+:::
+
```

Comment:
Would have thought that this empty line will break the unordered list?

Reply:
It does, but the figure does too so it doesn't really matter. 

![image](https://github.com/user-attachments/assets/0429d48b-ceaf-4176-896d-f521ad07868c)

Alternatively, we could move the figure to be below the list